pkg/api/aim/query/clause.go

```diff
@@ -15,24 +14,41 @@ type Regexp struct {
 
 // Build builds positive statement.
 func (regexp Regexp) Build(builder clause.Builder) {
-	builder.WriteQuoted(regexp.Column)
+	regexp.writeColumn(builder)
 	switch regexp.Dialector {
 	case postgres.Dialector{}.Name():
+		// #nosec G104
 		builder.WriteString(" ~ ")
 	default:
-		builder.WriteString(" regexp ")
+		// #nosec G104
+		builder.WriteString(" REGEXP ")
 	}
-	builder.AddVar(builder, fmt.Sprintf("%s", regexp.Value))
+	builder.AddVar(builder, regexp.Value)
 }
 
 // NegationBuild builds negative statement.
 func (regexp Regexp) NegationBuild(builder clause.Builder) {
-	builder.WriteQuoted(regexp.Column)
+	regexp.writeColumn(builder)
 	switch regexp.Dialector {
 	case postgres.Dialector{}.Name():
+		// #nosec G104
 		builder.WriteString(" !~ ")
 	default:
-		builder.WriteString(" NOT regexp ")
+		// #nosec G104
+		builder.WriteString(" NOT REGEXP ")
 	}
 	builder.AddVar(builder, regexp.Value)
 }
+
+func (regexp Regexp) writeColumn(builder clause.Builder) {
+	switch regexp.Dialector {
+	case sqlite.Dialector{}.Name():
+		// #nosec G104
+		builder.WriteString("IFNULL(")
```

Comment:
so you casting `null` to be an empty string and only then apply `regexp`. ok.

Reply:
correct